tests/test_logistic.py

```diff
@@ -137,3 +138,55 @@ def test_logistic_mixture_normalization():
     assert normalized == LogisticMixture(
         [Logistic(0.1, 0.01), Logistic(1, 0.1)], [0.5, 0.5]
     )
+
+
+def make_histogram(rv, xmin, xmax, num_bins):
```

Comment:
arguably this doesn't really belong here; I'm usually less strict with test code though so I think maybe it's fine